calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml

```diff
@@ -33,6 +33,9 @@ spec:
       - 166.9.28.71/32
       - 166.9.30.53/32
       - 166.9.32.51/32
+      - 166.9.94.31/32
```

Comment:
Are these going to be added to https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints ?

Reply:
Also, these, and the IPs below should also be added to https://github.com/IBM-Cloud/kube-samples/blob/master/calico-policies/private-network-isolation/eu-es/allow-private-services.yaml

Reply:
yes they're from test.cloud

Reply:
https://test.cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints lists

166.9.36.71
166.9.34.56 

as the eu-es endpoints

Reply:
Confirmed that the IPs in the docs are incorrect, and the ones in this PR are correct